modules/base/rendering/renderableplane.cpp

```diff
@@ -136,6 +174,10 @@ RenderablePlane::RenderablePlane(const ghoul::Dictionary& dictionary)
     , _mirrorBackside(MirrorBacksideInfo, false)
     , _size(SizeInfo, glm::vec2(10.f), glm::vec2(0.f), glm::vec2(1e25f))
     , _autoScale(AutoScaleInfo, false)
+    , _scaleByDistance(ScaleByDistanceInfo, false)
+    , _scaleRatio(ScaleRatioInfo, 0.01f)
+    , _scaleByDistanceMaxHeight(ScaleByDistanceMaxHeightInfo, 200000.f)
+    , _scaleByDistanceMinHeight(ScaleByDistanceMinHeightInfo, 30000.f)
```

Comment:
These properties need min and max values for the inputs in the UI to be useful

Reply:
Also, the min and max for the max and min height should be something more general. These values are a lot higher than the default size, for example. 

Maybe a min of 1 and a max of 10?